resources/assets/js/designer/components/modals/modal-create-report-add.vue

```diff
@@ -0,0 +1,330 @@
+<template>
+  <b-modal ref="modal" size="lg" @hidden="onHidden" centered title="Create Report Table">
+      <form>
+        <div class="form-group">
+          <label for="tableName" v-model="tableName">{{tableName}}</label>
+          <input type="text" class="form-control" id="tableName">
+        </div>
+
+        <div class="form-group">
+          <label for="description">{{description}}</label>
+          <textarea class="form-control" id="description" rows="3"></textarea>
+        </div>
+
+        <div class="d-flex justify-content-between">
+          <div class="form-group">
+            <label for="type" v-model="type">{{type}}</label>
+            <select class="form-control input-and-select" id="type" v-model="typeSelect">
+            <option v-for="select in typeSelectOptions">{{select}}</option>
+            </select>
+          </div>
+
+          <div class="form-group">
+            <label for="dbConnection" v-model="dbConnection">{{dbConnection}}</label>
+            <select class="form-control input-and-select" id="dbConnection" v-model="dbConnectionSelect">
+            <option v-for="select in dbConnectionSelectOptions">{{select}}</option>
+            </select>
+          </div>
+        </div>
+
+        <div>
+          <div class="d-flex justify-content-between">
+            <draggable class="drag-to-table-wrapper" v-model="list" :options="{group:'forms'}">
+                <div class="drag-to-table" v-for="form in list">
+                    <i class="fas fa-ellipsis-v fa-sm"></i><i class="fas fa-ellipsis-v fa-sm"></i>
+                  {{form.form_field_full}}
+                </div>
+            </draggable>
+
+            <i class="seperator fas fa-arrow-right"></i>
+
+            <div class="table-wrapper">
+              <div class="d-flex head-wrapper">
+                <div class="table-head form-field">form FIELD</div>
+                <div class="table-head name-field">NAME</div>
+                <div class="table-head label-field">LABEL</div>
+                <div class="table-head type-field">TYPE</div>
+                <div class="table-head size-field">size</div>
```

Comment:
Why is size and index lowercase?

Reply:
copy and paste from zeplin, they have been changed.